### PR TITLE
BXMSDOC-6600-master: Adopt SME feedback

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/installation/clustering-con.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [id='clustering-con_{context}']
 = {PRODUCT} clusters
 
-By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/installation/clustering-con.adoc:4]: "High availability decreases the chance of a loss of data when a single computer fails."

I suggest a minor change for minimalism. Totally up to you. I could be wrong..

High availability decreases the chance data loss when a single computer fails.

File: doc-content/enterprise-only/installation/clustering-dev-con.adoc [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
-====
-Configuration of {CENTRAL} for high availability is currently technology preview.
-====
-
 ifdef::PAM[]
 Developers can use {PRODUCT} to author rules and processes that assist users with decision making.
 
 endif::[]
-You can configure {PRODUCT} as a clustered development environment to benefit from high availability. With a clustered environment, if a developer is working on `$node1` and that node fails, that developer's work is preserved and visible on any other node of the cluster.
+You can configure {PRODUCT} as a clustered development environment to benefit from high availability. With a clustered environment, if a developer is working on the first node and that node fails, that developer's work is preserved and visible on any other node of the cluster.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/installation/clustering-dev-con.adoc:8]: Again, up to you. I suggest minor changes for:

"if a developer is working on the first node and that node fails, that developer's work is preserved and visible on any other node of the cluster."

if a developer is working a node that fails, the developer's work is preserved and visible on any other node in the cluster.

File: doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc [hunk 3]
```diff
@@ -7,7 +7,7 @@ With {PRODUCT}, you can set up a development environment to develop business app
...
+You can also cluster both development and runtime environments. A clustered development or runtime environment consists of a unified group or "cluster" of two or more servers. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration, while the primary benefit of clustering {PRODUCT} runtime environments is high availability and load balancing. High availability decreases the chance of a loss of data when a single server fails. When a server fails, another server fills the gap by providing a copy of the data that was on the failed server. When the failed server comes online again, it resumes its place in the cluster.
```
Comment (michelehaglund) [inline doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc:10]: Again, I would change the instance of "a loss of data" to "data loss". Up to you.

**Review by michelehaglund — APPROVED** [review]
Looks good @csherrar Minor comments. No show stoppers.

File: doc-content/enterprise-only/installation/clustering-con.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-con.adoc:4]: Done, will change to "High availability decreases the chance of data loss when a single computer fails."

File: doc-content/enterprise-only/installation/clustering-dev-con.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-dev-con.adoc:8]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/installation-options-ref.adoc:10]: Done

**Pull request merged**